### DISCUSSION TITLE
Live Grep Dotfiles

### DISCUSSION THREAD

**Discussion by jrock2004** [LazyVim/LazyVim#852] [Q&A]
When I go `<leader> sg` it will only search files where they do not start with a dot. How can I extend to include dotfiles?

**Comment by johannhmnn** [comment]
Hi! Adding this to `telescope.nvim` should work:

```lua
{
  "telescope.nvim",
  opts = {
    -- your telescope config
    pickers = {
      live_grep = {
        find_command = { "rg", "--hidden" },
      },
    },
  },
}
```

Also, you can take a look at [ripgrep's documentation](https://github.com/BurntSushi/ripgrep/blob/master/GUIDE.md) for possible arguments

**Reply by jrock2004** [reply]
Tried the following and still not working. But I did try this command line and it works fine

```
return {
  "nvim-telescope/telescope.nvim",
  opts = {
    keys = {
      { "<leader>,", false },
    },
    pickers = {
      live_grep = {
        find_command = { "rg", "--hidden" },
      },
    },
  },
}
```
